src/Security/InheritedPermissionFlusher.php

```diff
@@ -98,7 +98,7 @@ protected function getMemberIDList()
         }
 
         if ($this->owner instanceof Group) {
-            return $this->owner->Members()->column('ID');
+            return array_values($this->owner->Members()->getIDList());
```

Comment:
The PHPDoc says "Get a list of member IDs that need their permissions flushed"
It doesn't matter what order that happens in.